testapps/php56/tests/PHP56Test.php

```diff
@@ -53,13 +53,12 @@ public function testNonExistentPath()
 
     public function testPhpInfo()
     {
-        // Access to phpinfo.php, while phpinfo() is disabled by default.
+        // Access to phpinfo.php, while phpinfo() should be enabled by default.
         $resp = $this->client->get('phpinfo.php');
         $this->assertEquals('200', $resp->getStatusCode(),
                             'phpinfo.php status code');
-        $this->assertEquals('', $resp->getBody()->getContents(),
-                            'phpinfo() should be disabled and the content'
-                            . ' should be empty.');
+        $this->assertTrue(strlen($resp->getBody()->getContents()) > 1000,
```

Comment:
Why is `phpinfo` enabled by default when suhosin is active? It's included in the blacklist above.